website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -2789,13 +2789,14 @@ Status code `204`.
 
 The GET endpoints above can optionally return related resources, if requested with [the `include` query parameter](/terraform/cloud-docs/api-docs#inclusion-of-related-resources). The following resource types are available:
 
-* `organization` - The full organization record.
-* `current_configuration_version` - The last configuration this workspace received, excluding plan-only configurations. Terraform uses this configuration for new runs, unless you provide a different one.
 * `current_configuration_version.ingress_attributes` - The commit information for the current configuration version.
-* `current_run` - Additional information about the current run.
-* `current_run.plan` - The plan used in the current run.
-* `current_run.configuration_version` - The configuration used in the current run.
+* `current_configuration_version` - The last configuration this workspace received, excluding plan-only configurations. Terraform uses this configuration for new runs, unless you provide a different one.
 * `current_run.configuration_version.ingress_attributes` - The commit information used in the current run.
+* `current_run.configuration_version` - The configuration used in the current run.
+* `current_run.plan` - The plan used in the current run.
+* `current_run` - Additional information about the current run.
 * `locked_by` - The user, team, or run responsible for locking the workspace, if the workspace is currently locked.
-* `readme` - The most recent workspace README.md
+* `organization` - The full organization record.
```

Comment:
It seems like we flipped this whole table in reverse hierarchy order? Is there a reason we did that? I feel it reads less intuitively this way.

Reply:
So, I sorted it alphabetically because in the existing order, where would I place `project`. Is it as "important" as organization? or less important than "outputs"

Although I do agree with you in retrospect, the alpha order isn't necessarily correct.  How about:

```
current_configuration_version
current_configuration_version.ingress_attributes
current_run
current_run.configuration_version
current_run.configuration_version.ingress_attributes
current_run.plan
locked_by
organization
outputs
project
readme
```

I kind of wish we could indent to show hierarchy, but we don't really have prior art for that.

Reply:
Sounds good to me! 👍

Reply:
Done!

Reply:
Looks great; thank you! ✅